acceptance/lib/puppet/acceptance/install_utils.rb

```diff
@@ -125,7 +125,7 @@ def install_repos_on(host, project, sha, repo_configs_dir)
         project         = sha == 'nightly' ? project + '-latest'                :  project
         sha             = sha == 'nightly' ? nil                                :  sha
 
-        if sha == 'nightly'
+        if sha.nil?
```

Comment:
On line 126, we're actually overwriting `sha` with `nil` if it has been set to `nightly`